src/Database/Element/Column.php

```diff
@@ -28,6 +28,7 @@ class Column
     const TYPE_DATE = 'date';
     const TYPE_TIME = 'time';
     const TYPE_DATETIME = 'datetime';
+    const TYPE_DATETIME_TZ = 'timestamptz';
```

Comment:
If constant is called DATETIME, I would like the value be also datetime. Then just remap it in query builder / adapter

Reply:
Or name the constant TYPE_TIMESTAMP_TZ